### DISCUSSION TITLE
Ephemeral Buttons

### DISCUSSION THREAD

**Discussion by Legit4K** [discordjs/discord.js#7524] [Ideas]
Ephemeral messages exist so what about buttons, they would be useful too:

• can help reduce clutter in code by eliminating the need for verifying the interaction user matches expected target

• it can also provide an alternative to disabling buttons

Just a thought 💭

**Comment by suneettipirneni** [comment]
This isn't in the control of discord.js we're simply a wrapper for the API, we can't extend capabilities beyond what the API can do.

That being said, you can put button in ephemeral messages making the buttons themselves ephemeral.
